File: Codeforces/Python/136A-Presents/solution.py
# https://codeforces.com/problemset/problem/136/A

n = int(input())
give = list(map(int, input().split()))

receive = [0] * n

for idx, give_num in enumerate(give):
    receive[give_num - 1] = idx + 1

print(*receive)

File: Codeforces/C++/136A-Presents/solution.cpp
// https://codeforces.com/problemset/problem/136/A

#include <iostream>
#include <vector>

int main(int argc, char const *argv[])
{
    int n;
    std::cin >> n;

    std::vector<int> receive(n);

    for (int idx = 0; idx < n; idx++)
    {
        int give;
        std::cin >> give;

        receive[give - 1] = idx + 1;
    }

    for (int idx = 0; idx < n; idx++)
    {
        std::cout << receive[idx] << " ";
    }
    std::cout << "\n";
    return 0;
}
